Resources/Prototypes/Entities/Objects/Power/lights.yml

```diff
@@ -190,7 +190,8 @@
   parent: BaseLightTube
   name: exterior light tube
   description: A high power high energy bulb for the depths of space. May contain mercury.
-  id: ExteriorLightTube
+  id: ExteriorLightTubeBlue
```

Comment:
Why is the id changing?

Reply:
I'm already pushing. No idea why it did that